packages/kbn-es/src/cli_commands/source.js

```diff
@@ -42,9 +45,14 @@ exports.run = async (defaults = {}) => {
       installPath: 'install-path',
       sourcePath: 'source-path',
       dataArchive: 'data-archive',
+      skipReadyCheck: 'no-ready-check',
```

Comment:
Testing this locally, it looks like there's no way to set this to true?  `getopts` seems to be interpreting this in the positive `ready-check` form.

Reply:
Oh, thank you for testing that, I'll switch to a `--skip` prefix

Reply:
Updated